cmd/kubeadm/app/cmd/join.go

```diff
@@ -61,28 +61,66 @@ func NewCmdJoin(out io.Writer) *cobra.Command {
 	var cfgPath string
 
 	cmd := &cobra.Command{
-		Use:   "join <master address>",
+		Use:   "join <args> [API server]",
```

Comment:
Our errors refer to this as `DiscoveryTokenAPIServers`.  That is a mouthful.  Not sure how to pull this through.  Better to be explicit than not.  I'd line up the strings so that if people see the error and then trigger help they can figure it out.